Hazel/src/Hazel/Renderer/Renderer2D.cpp

```diff
@@ -219,6 +219,7 @@ namespace Hazel {
 		const float textureIndex = 0.0f; // White Texture
 		constexpr glm::vec2 textureCoords[] = { { 0.0f, 0.0f }, { 1.0f, 0.0f }, { 1.0f, 1.0f }, { 0.0f, 1.0f } };
 		const float tilingFactor = 1.0f;
+		const glm::vec2 uvOffsetFactor = { 0.0f, 0.0f };
```

Comment:
In the other DrawQuad function you called it `texture_uvoffset` and here you call it `uvOffsetFactor`. Unify them to one and the same name. I suggest `textureUVOffset` as this is what you used in the YAML functions.